samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -82,17 +82,19 @@
    * non-stream systems (e.g., an external database).
    *
    * @param sinkFn the function to send messages in this stream to an external system
+   * @return this {@link MessageStream} object
```

Comment:
This means before the external stream. For consuming from the external stream, it should either be partitionBy or sendThrough.